Resources/Textures/_RMC14/Objects/Misc/target.yml

```diff
@@ -0,0 +1,83 @@
+- type: entity
+  name: human target
+  id: RMCTargetHuman
+  parent: BaseStructureDynamic
+  abstract: true
+  description: A shooting target. This one is a Human.
+  suffix: RMC
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Security/target.rsi
+    state: target_h
+    noRot: true
+  - type: Repairable
+  - type: DamagePopup
+    allowTypeChange: true
+    damagePopupType: Combined
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 200
+        mask:
+        - FullTileMask
+        layer:
+        - MobLayer
+  - type: InteractionOutline
+  - type: Physics
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100000000000000000000
+      behaviors:
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 500000000000000000000
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          CMSteel:
+            min: 5
+            max: 5
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```

Comment:
If you don't want these to trigger you can remove the destructible component
```suggestion
```

Reply:
Having it be destructible is rather funny just in case people make a team effort to break it. I left that in as I found it amusing.